R/timeseriessummary-data.R

```diff
@@ -388,4 +389,86 @@ parseTSSGapTolerances <- function(reportData, timezone){
   }
   
   return(gapTolerances)
-}
\ No newline at end of file
+}
+
+#' Format TSS Correction Parameters
+#' @description formats the parameters so they are prettier to print
+#' @param corrections the corrections for the report
+#' @param timezone the timezone to parse data into
+#' @return corrections with formattedParameters field to print in report
+formatCorrectionParameters <- function(corrections, timezone) {
+  
+  corrections[["formattedParameters"]] <- ""
+  
+  if(!isEmptyOrBlank(corrections) && nrow(corrections)>0) {
+    
+    for (i in seq(nrow(corrections))) {
```

Comment:
Yeah, I redid this like three times, but the data come through oddly and this seemed to work alright ...